content/handbook/support/workflows/500_errors.md

```
@@ -0,0 +1,124 @@
+
+[Sentry ワークフロー](/handbook/support/workflows/sentry) を参照してください。
+
+Kibana と Sentry を使った 500 エラー調査のビデオウォークスルーは、[ここ](https://youtu.be/o02t3V3vHMs) （GitLab Unfiltered）で確認できます。
+
+#### 結果を Issue にする
+
+Kibana か Sentry で結果が見つかったら、以下を行います。
+
+1. できるだけ多くの情報を集めます。Kibana か Sentry で見つかったログへのリンクを含む内部ノートをチケットに作成します。
+1. [GitLab Issue トラッカー](https://gitlab.com/gitlab-org/gitlab) で重複または関連する Issue を検索します。
+1. Issue が既知か未知かを確認し、それに従って進めます: [Issue が既知の場合](#issue-is-known) または [Issue が未知の場合](#issue-is-unknown)。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)